template.go

```diff
@@ -1,293 +1,137 @@
-            <table class="main" width="100%" cellpadding="0" cellspacing="0">
-              <tr>
-                {{#success build.status}}
-                  <td class="alert alert-good">
-                    <a href="{{ system.link_url }}/{{ repo.owner }}/{{ repo.name }}/{{ build.number }}">
-                      Successful build #{{ build.number }}
-                    </a>
-                  </td>
-                {{else}}
-                  <td class="alert alert-bad">
-                    <a href="{{ system.link_url }}/{{ repo.owner }}/{{ repo.name }}/{{ build.number }}">
-                      Failed build #{{ build.number }}
-                    </a>
-                  </td>
-                {{/success}}
-              </tr>
-              <tr>
-                <td class="content-wrap">
-                  <table width="100%" cellpadding="0" cellspacing="0">
-                    <tr>
-                      <td>
-                        Repo:
-                      </td>
-                      <td>
-                        {{ repo.owner }}/{{ repo.name }}
-                      </td>
-                    </tr>
-                    <tr>
-                      <td>
-                        Author:
-                      </td>
-                      <td>
-                        {{ build.author }}
-                      </td>
-                    </tr>
-                    <tr>
-                      <td>
-                        Branch:
-                      </td>
-                      <td>
-                        {{ build.branch }}
-                      </td>
-                    </tr>
-                    <tr>
-                      <td>
-                        Commit:
-                      </td>
-                      <td>
-                        {{ truncate build.commit 8 }}
-                      </td>
-                    </tr>
-                    <tr>
-                      <td>
-                        Time:
-                      </td>
-                      <td>
-                        {{ duration build.started_at build.finished_at }}
-                      </td>
-                    </tr>
-                  </table>
-                  <hr>
-                  <table width="100%" cellpadding="0" cellspacing="0">
-                    <tr>
-                      <td>
-                        {{ build.message }}
-                      </td>
-                    </tr>
-                  </table>
-                </td>
-              </tr>
-            </table>
-          </div>
-        </td>
-        <td></td>
-      </tr>
-    </table>
-  </body>
-</html>
-`
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+	"unicode"
+	"unicode/utf8"
+
+	"github.com/aymerick/raymond"
+)
+
+func init() {
+	raymond.RegisterHelpers(funcs)
+}
+
+// Render parses and executes a template, returning the results in string format.
+func Render(template string, payload interface{}) (s string, err error) {
```

Comment:
Please vendor and use https://github.com/drone/drone-go/blob/master/template/template.go#L21 instead of reinventing the wheel.